Header header
float64 prim_cost
float64 speed
float64 latency
int16 follow_prim